packages/proxy-target/contracts/HifiProxyTarget.sol

```diff
@@ -565,13 +565,13 @@ contract HifiProxyTarget is IHifiProxyTarget {
         uint256 depositAmount,
         uint256 underlyingOffered
     ) public override {
-        // When the underlying moonlights as the collateral, the user can borrow on a one-to-one basis.
-        uint256 maxBorrowAmount = normalize(depositAmount, hifiPool.underlyingPrecisionScalar());
+        // When the underlying moonlights as the collateral, the user can mint on a one-to-one basis.
+        uint256 maxHtokenMinted = normalize(depositAmount, hifiPool.underlyingPrecisionScalar());
```

Comment:
Needs to be renamed to "hTokenMinted", since the amount minted can only be the exact equivalent to the amount deposited.

Reply:
I preferred to name it maxHtokenMinted because not In all cases  `hTokenRequired` will be equal to hTokens minted. Since, hTokens minted is greater or equal than `hTokenRequired`, because not all of `maxHtokenMinted`  will be used to add liquidity to pool. `depositAmount` will determine maximum  hToken that can be minted by depositing the underlying in the HToken contract to mint hTokens but `hTokenRequired` to add liquidity in pool depends on  `underlyingOffered`  and so,` if (maxHtokenMinted > hTokenRequired)`  remaining  hTokens minted will be relayed back to the user.

But yes, feel free to rename it if it seems more correct.